producer-rest-core/src/main/scala/za/co/absa/spline/producer/rest/controller/v1/package.scala

```diff
@@ -14,10 +14,10 @@
  * limitations under the License.
  */
 
-package za.co.absa.spline.swagger
+package za.co.absa.spline.producer.rest.controller
 
-import java.io.File
+package object v1 {
 
-case class SwaggerDocGenConfig(
-  maybeOutputFile: Option[File] = None,
-  restContextClass: Option[Class[_]] = None)
+  trait _package
```

Comment:
What is the purpose of this trait?

Reply:
It's just an anchor type that is use to reference the package